Formula/gauge.rb

```diff
@@ -42,6 +42,12 @@ def install
     end
   end
 
+  def caveats; <<-EOS.undent
```

Comment:
@sriv see https://github.com/Homebrew/brew/blob/c2cbfc1526a0bbb50683e057ceed0210cef06e83/Library/Homebrew/formula.rb#L984-L986. The information about gauge's telemetry doesn't really meet those criteria.